cmd/createorupdate/createorupdate.go

```diff
@@ -117,21 +116,16 @@ func execute(
 	conf *fakerp.Config,
 	adminManifest string,
 ) (*v20180930preview.OpenShiftManagedCluster, error) {
-	dataDir, err := shared.FindDirectory(shared.DataDirectory)
-	if err != nil {
-		return nil, fmt.Errorf("failed to find %s: %v", shared.DataDirectory, err)
-	}
-
 	if adminManifest != "" {
 		oc, err := fakerp.GenerateManifestAdmin(adminManifest)
 		if err != nil {
 			return nil, fmt.Errorf("failed reading admin manifest: %v", err)
 		}
-		defaultAdminManifest := filepath.Join(dataDir, "manifest-admin.yaml")
```

Comment:
We had all this dataDir logic because when executed from `./hack/upgrade.sh` or called from `pkg/tests/...` it was under the different relative path and was not able to find files. If I remember it was keyrotation tests. Is this not a case anymore?

Reply:
The whole thing was quite jumbled.  In general, where possible, test should work with external representation cs fetched from the RP with a GET call.  The only valid exception to this currently is the internal representation cs is needed to build the authenticators (relies on seeing the certs, which are not end-user visible).

So it boils down to these cases:
1. test needs an external cs - make a GET call
2. e2e test framework needs internal cs, is running via go test.  Our cwd will be <root>/test/e2e because that's where `func TestE2E(t *testing.T)` is.  Look in `../../_data/containerservice.yaml`.
3. e2e test framework needs internal cs, is running via compiled test binary.  Our cwd will be <root>.  Look in `_data/containerservice.yaml`.
4. fakerp.  Our cwd will be <root>.  Look in `_data/containerservice.yaml`.